framework/src/base/MooseApp.C

```diff
@@ -344,7 +330,7 @@ void
 MooseApp::meshOnly(std::string mesh_file_name)
 {
   /**
-   * These actions should be the minimum set necessary to generate and output
+   * Thesgete actions should be the minimum set necessary to generate and output
```

Comment:
You might just fix this later. I'm going to merge since everything else looks good.